src/dropin.js

```diff
@@ -2,19 +2,21 @@
 
 var assign = require('./lib/assign').assign;
 var analytics = require('./lib/analytics');
-var MainView = require('./views/main-view');
 var constants = require('./constants');
 var DropinError = require('./lib/dropin-error');
 var DropinModel = require('./dropin-model');
 var EventEmitter = require('./lib/event-emitter');
 var isGuestCheckout = require('./lib/is-guest-checkout');
 var fs = require('fs');
+var MainView = require('./views/main-view');
+var paymentOptionIDs = constants.paymentOptionIDs;
 var translations = require('./translations');
 var uuid = require('./lib/uuid');
 
 var mainHTML = fs.readFileSync(__dirname + '/html/main.html', 'utf8');
 var svgHTML = fs.readFileSync(__dirname + '/html/svgs.html', 'utf8');
 
+var DEFAULT_LOG_LEVEL = 'warn';
```

Comment:
Should we specify this as `DEFAULT_CHECKOUTJS_LOG_LEVEL` since it's not scoped to the paypal file anymore?